flytekit/remote/remote.py

```diff
@@ -667,6 +673,13 @@ def _execute(
 
         with self.remote_context() as ctx:
             input_python_types = entity.guessed_python_interface.inputs
+            expected_input = entity.interface.inputs
+            print("fuck", expected_input)
```

Comment:
remove?

Reply:
sorry, I was testing something. forgot to remove it.